e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -265,6 +265,69 @@ test.describe('Notebook entry tests', () => {
     });
     test.fixme('new entries persist through navigation events without save', async ({ page }) => {});
     test.fixme('previous and new entries can be deleted', async ({ page }) => {});
+    test('when a valid link is entered into a notebook entry, it becomes clickable when viewing', async ({ page }) => {
+        const TEST_LINK = 'http://www.google.com';
```

Comment:
http? What is this? 2017?

Reply:
It stands for Hyper Text Transfer Protocol. It was created by Tim Berners-Lee.

In 1991.

Duh.